packages/lit-dev-content/site/tutorials/content/pointer-events/01.md

```diff
@@ -0,0 +1,23 @@
+Pointer Events are modelled after Mouse Events, and therefore have very similar naming.
+
+  * `pointerdown`
+    * Fired when a finger or pen touches the screen, or a mouse button is pressed.
+  * `pointerup`
+    * Fired when a finger or pen leaves the screen, or a mouse button is released.
+  * `pointermove`
+    * Fired when the mouse moves, or a finger or pen touching the screen moves.
+  * `pointerover`
+    * Fired when a pointer moves into an element.
+  * `pointerout`
+    * Fired when a pointer moves out of an element.
+  * `pointerenter`
+    * Similar to `pointerover`, but also fires for the element's descendants, and does not bubble.
+  * `pointerleave`
+    * Similar to `pointerout`, but also fires for the element's descendants, and does not bubble.
+
+However, since Pointer Events are also used for touch, there are several more events that relate to touch-specific scenarios.
+
+  * `pointercancel`
+    * This event is fired if the browser decides that a touch interaction should scroll instead. Once this is fired, no more pointer events will fire for this interaction.
+  * `gotpointercapture` and `lostpointercapture`
+    * These events relate to [Pointer Capturing](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events#capturing_the_pointer), which changes where Pointer Events are targeted, and is covered in a later section.
```

Comment:
Can add a small section that describes what's happening on the right? Something like

> In the example we... handle events with lit's `@` event (link to docs) and log XYZ

Also I'm not quite sure what's going on in the "solve" example mind explaining what's going on there too. Or until I add the feature to remove the "solve" button for some steps, just copy paste the before step into the after step